justfile

```diff
@@ -11,8 +11,8 @@ export COMPILE := BIN + "/pip-compile --allow-unsafe"
 
 
 # list available commands
-default:
-    @{{ just_executable() }} --list
+_default:
+    @just --list
```

Comment:
question: Why were these types of changes in the `justfile` required at this point? Were they required for making the Windows tests work, or are they incidental changes?

(Why I ask is that the repo template [currently uses `just_executable()`](https://github.com/opensafely-core/repo-template/blob/e9b08d4a8ce7836970955ea5309592b986bd72cf/justfile#L10-L11) and I assume that this feature of Justfiles works cross-platform.)

Reply:
So, it doesn't seem to work cross platform.

On windows, `{{ just_executable() }}` returns `C:\path\to\just.exe`, i.e. with backslashes. `sh` understandably doesn't like this.

We know just is on the path in both linux and windows, so using it directly seems to work fine. In scenarios where just was wasn't on the path, this approach wouldn't work.

Reply:
If you want the original incantation to work maybe you can surround it in quotes so the single backslashes won't disappear on Windows, i.e.,
```just
default:
    @"{{ just_executable() }}" --list
```

Reply:
Yeah, I wondered if that may work, but I think I prefer using plain `just`.

{{ just_executable() }} is a solution to a problem I've not encountered yet.

Reply:
Also, I think it would need to be double backslashed \\ ?